cfme/tests/ansible/test_embedded_ansible_manual.py

```diff
@@ -361,3 +361,26 @@ def test_embed_tower_playbook_with_retry_method():
             9. Check automation.log to make sure the playbook retried 3 times and then ended OK.
     """
     pass
+
+
+@pytest.mark.tier(3)
+@pytest.mark.meta(coverage=[1807928])
+def test_embed_tower_exec_play_with_diff_auth(appliance, provider):
+    """
+    User/Admin is able to execute playbook from git private repo with
+    git submodule in the bare repo and test this as per this scenarios,
+    - A playbook git repo over https without auth (with password)
+    - A playbook git repo over https with auth (with access token)
+    - A playbook git repo over ssh-key
```

Comment:
If this is something you're going to parametrize when you automate the test, I suggest you add those parameters here itself, meaning to say, parametrize this manual test.

Reply:
DONE